packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: OIN submission requirements
+meta:
+  - name: description
+    content: Use this guide to prepare all the requirements prior to submitting your integration to Okta for publication in the Okta Integration Network.
+layout: Guides
+---
+
+This guide provides you with a list of requirements you need to prepare prior to submitting your integration for publication in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/).
+
+---
+
+**Learning outcome**
+
+* Understand the requirements necessary for the submission process using the [OIN Manager](https://oinmanager.okta.com).
```

Comment:
Maybe...

"... necessary to submit your integration using the OIN Manager..."?